src/cephadm/cephadm.py

```diff
@@ -6185,6 +6186,7 @@ def command_deploy(ctx):
         c = get_deployment_container(ctx, ctx.fsid, daemon_type, daemon_id)
         deploy_daemon(ctx, ctx.fsid, daemon_type, daemon_id, c, uid, gid,
                       reconfig=ctx.reconfig,
+                      redeploy=redeploy,
```

Comment:
```suggestion
                      reconfig=reconfig,
                      redeploy=redeploy,
```

(nit) 

Using `ctx.reconfig` (instead of `reconfig`), while also using a `redeploy` var (instead of `ctx.redeploy`) is a bit confusing and potentially error prone.

Maybe this is a good opportunity to clean these up by defining `reconfig = ctx.reconfig` at the top of the function where the `redeploy` var is defined?

Reply:
done